mosaic-runtime/src/main/kotlin/com/jakewharton/mosaic/mosaic.kt

```diff
@@ -59,16 +68,82 @@ fun runMosaic(body: suspend MosaicScope.() -> Unit) = runBlocking {
 		}
 	}
 
+	lateinit var terminal: Terminal
+	lateinit var terminalInfo: MutableState<TerminalInfo>
+	terminal = TerminalBuilder.builder()
+		.nativeSignals(true)
+		.signalHandler {
+			when (it) {
+				WINCH -> {
+					terminalInfo.value = TerminalInfo(
+						Size(
+							width = terminal.width,
+							height = terminal.height,
+						)
+					)
+				}
+				else -> {}
+			}
+		}
+		.build()
+
+	terminalInfo = mutableStateOf(
+		TerminalInfo(
+			Size(
+				width = terminal.width,
+				height = terminal.height,
+			)
+		)
+	)
+
+	terminal.enterRawMode()
+	val terminalReader = terminal.reader()
+	val keyHandlers = mutableListOf<(KeyEvent) -> Unit>()
+	launch(context = composeContext + Dispatchers.IO) {
+		while (isActive) {
+			val keyEvent = when (terminalReader.read()) {
+				'q'.code -> KeyEvent.Q
+				27 -> {
+					when (terminalReader.read()) {
+						91 -> {
```

Comment:
on Windows `cmd` this is `79 ->`
```suggestion
						91, 79 -> {
```

Reply:
https://github.com/JakeWharton/mosaic/pull/230/files